tozny/resource_account.go

```diff
@@ -132,6 +131,13 @@ func resourceAccount() *schema.Resource {
 		ReadContext:   resourceAccountRead,
 		DeleteContext: resourceAccountDelete,
 		Schema: map[string]*schema.Schema{
+			"persist_credentials_to": {
+				Description: "Where to persist the generated account credentials. Default: none, they are not persisted.",
```

Comment:
ValidateFunc function useful for enforcing valid enum value provided

https://www.terraform.io/docs/extend/schemas/schema-methods.html